mavros_msgs/msg/HilGPS.msg

```diff
@@ -0,0 +1,17 @@
+# HilControls.msg
+#
+# ROS representation of MAVLink HIL_GPS
+# See mavlink message documentation here:
+# https://pixhawk.ethz.ch/mavlink/#HIL_GPS
+
+std_msgs/Header header
+uint8 fix_type
+geographic_msgs/GeoPoint geo
```

Comment:
@TSC21 i suggested to make our `mavros_msgs/GeoPointInt`, not usage of geographinc-info.
Something like:
```
uint32 latitude_1e7
uint32 longitude_1e7
uint32 altitude_mm
```

That msg was smaller than two float64 + one 32, but usage of generic lla fields looks more intuitive.